platform-api/src/routers/forms.py

```
@@ -0,0 +1,125 @@
+from services.http_client import get_client
+from services.sanity_client import SanityClient
+from services.turnstile import verify_turnstile
+from dependencies import get_settings, verify_admin_api_key, get_sanity
+from datetime import datetime, timezone
+from utils.dataset import resolve_dataset
+import uuid
+
+router = APIRouter(prefix="/forms", tags=["forms"])
+
+@router.post("/submit", response_model=SubmissionResponse)
+async def submit_form(request: Request, body: FormSubmission, settings: WorkerSettings = Depends(get_settings), sanity = Depends(get_sanity)):
```

Ruff (0.15.11)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)